src/request.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
```

Comment:
don't need to create this file. You can test it open the server url on the browser